docs/intro/intro_scaling.rst

```diff
@@ -242,13 +246,12 @@ This gives us significant time savings
     In [1]: %timeit bin_packing(weights)
     95.5 ms ± 2.87 ms per loop (mean ± std. dev. of 7 runs, 10 loops each)
 
-Don't copy constraints
-======================
+Constraints
+===========
 
-By default :meth:`~dimod.ConstrainedQuadraticModel.add_constraint`
-create a copy of the objects given to it.
-However, in this case we are immediately discarding the models created in our
-function, so we can safely skip the copy step.
+By default, the :meth:`~dimod.ConstrainedQuadraticModel.add_constraint` method
+creates a copy of the objects you give it. If these objects are used solely for
```

Comment:
I think this can be a bit clearer. I think the point is that we're passing mutable objects in, but since we're never using them again it's safe for the function to mutate them.